docs/Basics/index.md

```diff
@@ -6,37 +6,38 @@ The variety of AEC (Architecture, Engineering and Construction) commercial softw
 
-By extension, the BHoM (Building and Habitats object Model) is the collection terminologies and of the functionality built on top of such terminology.
+By extension, the BHoM (Building and Habitats object Model) is the collection of these **object Models** and the functionality built on top of them.
 
 ## Linking Software
 
-Across the AEC industry, regardless of our discipline, we have to work with multiple softwares during the course of any given project. Since there is rarely a simple solution to transfer the data from one software to another, we usually end up either doing it manually each time or writing some bespoke script to automate the transfer. Things get even more complex when we work across disciplines and with other collaborators. When the number of software to deal with becomes more than just a few, this one-to-one translation solution quickly becomes intractable.
+Across the AEC industry, regardless of our discipline, we have to work with multiple softwares during the course of any given project. Since there is rarely a simple solution to transfer the data from one software to another, we usually end up either doing it manually each time or writing some bespoke script to automate the transfer. Things get even more complex when we work across disciplines and with other collaborators. When the number of programs to deal with becomes more than just a few, this one-to-one translation solution quickly becomes intractable.
 
-This is where the BHoM comes in. Thanks to the central common language, it is possible to interoperate between many different software. Instead of creating translators between every possible pair of software, you just have to write one single translator between BHoM and a target software, to then connect to all the others software. 
-We call those translators **Adapter**s.
+This is where the BHoM comes in. Thanks to the central common language, it is possible to interoperate between many different programs. Instead of creating translators between every possible pair of applications, you just have to write one single translator between BHoM and a target software, to then connect to all the others.
+
+We call those translators **Adapters**.
 
 ![](https://raw.githubusercontent.com/BHoM/documentation/main/Images/InteropA.png)
 
 
 
 ## Linking functionality
-Most often, AEC industry experts creates ad-hoc functions and tools when the need arises. Common examples are large, complex spreadsheets, VBA macros for Excel, but also small and large User Interface programs written in C#, Python or other means.
+Most often, AEC industry experts creates ad-hoc functions and tools when the need arises. Common examples are large, complex spreadsheets, VBA macros for Excel, but also small and large User Interface programs written in C#, Python or languages.
```

Comment:
```suggestion
Most often, AEC industry experts create ad-hoc functions and tools when the need arises. Common examples are large, complex spreadsheets, VBA macros for Excel, but also small and large User Interface programs written in C#, Python or other tools.
```

Scripts/ad-hoc tools are produced across the AEC industry in more than just programming languages.